example/tests/test_pii.py

```diff
@@ -12,6 +12,6 @@ def test_pii_checker_on_pii():
     assert len(results) == 1
 
     report = results[0]
-    assert report['check_pii_detail:pii-found:street_addresses'][2] == ['14 Naresuan Sq']
-    assert report['check_pii_detail:pii-found:ips'][2] == ['10.8.34.199', '127.0.34.199', '192.168.23.109']
+    assert report[0]['item']['entity']['location']['index'] == ['14 Naresuan Sq']
+    assert report[0]['item']['entity']['location']['index'] == ['10.8.34.199', '127.0.34.199', '192.168.23.109']
```

Comment:
Not 100% sure this would work - unless there's a fairly odd bug in the reporting? Can take a look with you?